docs/tutorials/stateful-application/cassandra.md

```diff
@@ -158,7 +158,7 @@ Use `kubectl edit` to modify the size of of a Cassandra StatefulSet.
         # and an empty file will abort the edit. If an error occurs while saving this file will be
         # reopened with the relevant failures.
         #
-        apiVersion: apps/v1beta2
+        apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
Line 171 should be changed.
if you change the API version, that `v1beta1` should be `v1` too.
of course original `v1beta1`  is wrong also.

Reply:
Good catch!

Reply:
Updated. Thanks!
@zhangxiaoyu-zidif